Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.8-alpine
```

Comment:
The Docker guidelines in our development guide [suggest using base images](https://github.com/18F/development-guide/tree/master/project_setup/docker#base-images) instead of slim or alpine images since they more closely resemble our production environment. I'll yield to @konklone on whether we need to do that here, but wanted to point it out.